packages/core/test/tabs/tabsTests.tsx

```diff
@@ -28,10 +28,9 @@ describe("<Tabs>", () => {
     // default tabs content is generated from these Dsin each test
     const TAB_IDS = ["first", "second", "third"];
 
-    // selectors using ARIA role
-    const TAB = "[role='tab']";
-    const TAB_LIST = "[role='tablist']";
-    const TAB_PANEL = "[role='tabpanel']";
+    const TAB_SELECTOR = `.${Classes.TAB}`;
```

Comment:
No other test grabs by role-- use `Classes` like the rest.

Reply:
I've always heard it's best to select elements in tests as close as possible as to how your users would. Ideally by actual text content, followed by accessibility roles/element types (ex user searches for and clicks a `<button>`), and as a last resort something like targeting by an `id` or special test-id attribute.

So I don't think we should change these tests just because other tests are selecting by class name.

Reply:
OK! Changed.